test/imported/test_indexing.py

```diff
@@ -27,7 +29,42 @@ def set_(reference: Tensor, shape, strides, offset):
   assert strided.lazydata in reference.lazydata.base.views, "base.views should contain strided.lazydata"
   return strided
 
-def make_tensor(shape, dtype:dtypes, noncontiguous):
+# TODO tries to mimic .detach().copy() or just .copy() behavior
+# for torch.copy() the resulting tensor.data_ptr() is different than the original
+# some random medium article says .clone() is a shallow copy, so maybe this is correct?
+# https://discuss.pytorch.org/t/clone-and-detach-in-v0-4-0/16861/4
+def clone(original:Tensor):
```

Comment:
we probably don't need clone #2628. because tinygrad is lazy. we might not want to follow some torch expectation (say they make a copy) if we don't need to